cpp/src/gandiva/precompiled/string_ops_test.cc

```diff
@@ -110,7 +110,7 @@ TEST(TestStringOps, TestCastVarhcar) {
   uint64_t ctx_ptr = reinterpret_cast<int64>(&ctx);
   int32 out_len = 0;
 
```

Comment:
Please add a test for `out_len < 0`. If I'm reading this right then it'll return an empty string. Is that expected, or should it emit an error?

Reply:
Emitting an error makes sense. I have updated it.